domain/domain.go

```diff
@@ -16,3 +18,11 @@ func HttpStatusFmt(code int) string {
 	}
 	return fmt.Sprintf("%d (%s)", code, txt)
 }
+
+// Equals performs a deep equals against two objects and
+// fails if they're not equal.
+func Equals(tb testing.TB, exp interface{}, got interface{}) {
+	if !reflect.DeepEqual(exp, got) {
+		tb.Fatalf("\texp: %#v\n\tgot: %#v", exp, got)
```

Comment:
Perhaps use the format we're using elsewhere that shows the value and the type?

Reply:
done